app/serializers/shipit/task_serializer.rb

```diff
@@ -6,7 +6,7 @@ class TaskSerializer < ActiveModel::Serializer
     has_one :author
     has_one :revision, serializer: ShortCommitSerializer
 
-    attributes(:id, :url, :html_url, :output_url, :type, :status, :action, :title, :description, :started_at, :ended_at, :updated_at, :created_at, :env, :ignored_safeties)
+    attributes(:id, :url, :html_url, :output_url, :type, :status, :action, :title, :description, :started_at, :ended_at, :updated_at, :created_at, :env, :ignored_safeties, :max_retries, :retry_attempt)
```

Comment:
For posterity: These changes came from https://github.com/Shopify/shipit-engine/pull/1111/.

@nishanthooda Another option is to keep the second PR around until the first one is merged, and then retarget to master. This _may_ require re-approvals from reviewers. But it also prevents "surprise" files in the first PR. YMMV, no approach is perfect.